src/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../styles/ZIndexes.scss';
+@import '../../styles/Colors.scss';
+@import '../../styles/Mixins.scss';
+
+.dropdown {
+  position: sticky;
+
+  &__opener {
+    position: relative;
+  }
+
+  &__button {
+    text-transform: none !important;
+    flex: 1 !important;
+  }
+
+  &__container {
+    background-color: $white;
+    z-index: $ZIndexDropdown;
+    box-shadow: 0px 0px 8px 1px rgba(0, 0, 0, 0.3);
+    position: absolute;
+    width: 150px;
+    top: 50px;
+    right: 20px;
+    padding: 10px 20px;
+    border-radius: 5px;
+
+    & li {
```

Comment:
1. & is not necessarry here if you don't need to create class like this 
dropdown__container__li
2. li in ul is not correct because ul is equal to unordered list and li is a list item
3. don't target by tags
4. structure is too deep for ul and li